upstream-seed/README.md

```diff
@@ -0,0 +1,16 @@
+# Upstream Seed
+
+A light-weight seed node that tracks and replicates the configured Radicle
+Projects.
+
+## Features
+
+* Radicle peer based on [`librad`][librad] that participates in gossip.
+* Tracks projects configured in the `--project` option. Only data
+  of the first remote that provides the project is replicated. Data of
```

Comment:
So this basically means that whoever first pushes the project onto the seed will be the one who's view will get replicated, correct? And it might not necessarily be the maintainer of the project.

Reply:
Yes, that’s correct. https://github.com/radicle-dev/radicle-upstream/issues/2569 tracks the fix for this.